src/navs/index.js

```diff
@@ -2336,6 +2336,16 @@ export const docsMenu = {
                     icon: 'IconToggle',
                     color: 'red',
                 },
+                {
+                    name: 'Mobile session replay',
+                    url: '/docs/session-replay/mobile',
+                    icon: 'IconPhone',
+                    color: 'blue',
+                    badge: {
+                        title: 'Beta',
+                        className: 'uppercase !bg-blue/10 !text-blue !dark:text-white !dark:bg-blue/50',
+                    },
+                },
```

Comment:
I dont think we need this anymore. Instead, I think it probably makes more sense to include the Wireframe vs screenshot mode info in the installation docs. WDYT?

Reply:
Yeah I feel this, now it's dangling. Though I want to actually preserve this and turn it into an explainer on how session replay works technically on web AND mobile, but I want to do that as a separate job. Can we live with the imperfection until then?

Reply:
This sounds good!